core/defaultconfigs/spamd.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+import os
+import kcconf
+
+# Component specific configurations
+kcconf.configkopano({
+    r"/tmp/kopano/spamd.cfg":
+    {
+        'log_file': "-",
+        'log_level': "3"
```

Comment:
For the other containers there are no container specific defaults any longer (see https://github.com/zokradonh/kopano-docker/blob/master/core/defaultconfigs/server.py for example). `log_level=3` would be the service default anyways. So to specify the log file location I want to ask you to do this in the compose file instead.